app/Resources/translations/default/AdminOrderscustomersNotification.xlf

```diff
@@ -929,11 +929,6 @@
         <target>Order #%d cannot be loaded.</target>
         <note>Line: 1408</note>
       </trans-unit>
-      <trans-unit id="3354cb02aba47cf9f44d703b25a0a777">
```

Comment:
Shouldn't we keep this ? Do BC breaks apply to translations ? What if a module uses this translation ?

Reply:
No don't worry this wording has been introduced in 177 only, so we can remove it
There are a ew others that need to be clean as well in another issue (currency's wording)